src/ossl/aes.rs

```diff
@@ -1144,6 +1204,7 @@ impl AesOperation {
         }
     }
 
+    /// Instantiates a messaged-based encryption operation
```

Comment:
```suggestion
    /// Instantiates a Message Based encryption operation
```
the same for the following docs entries. Using the version without the dash as this is used in the previous doc entries and would be good to have the naming systematic. Actually I do not care if it will be "message-based" or "Message Based", but it should be systematic.